xbmc/GUIInfoManager.cpp

```diff
@@ -3563,15 +3565,15 @@ const infomap container_ints[] = {{ "row",              CONTAINER_ROW },
 ///                  \anchor Container_Content_parameter
 ///                  _string_,
 ///     @return **True** if the current container you are in contains the following:
-///       - <b>files</b> 
-///       - <b>songs</b> 
+///       - <b>files</b>
+///       - <b>songs</b>
 ///       - <b>artists</b>
-///       - <b>albums</b> 
+///       - <b>albums</b>
```

Comment:
Revert cosmetic whitespace change....
You can see all of them. Fix it!!

Reply:
All fixed......I think.